homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/profile/header/AnimatedProfileHeader.kt

```diff
@@ -276,34 +290,70 @@ private fun AnimatedProfileHeaderSavedState(
     }
 }
 
+/*
+ * This is a static version of the header to measure its height with the minimal UI components that are needed the measure the height.
+ * We can use the real composable to measure the height of the header in the collapsed state because of the animations.
+ */
+@Composable
+private fun StaticCollapsedStateToMeasure(profile: Profile) {
```

Comment:
I know this is not ideal, however, we can't measure the real animated header because the height changes due to the animation. I'm open to a different approach if we find something better.